stories/modules-tagmanager-setup.stories.js

```diff
@@ -96,6 +96,15 @@ storiesOf( 'Tag Manager Module/Setup', module )
 
 		return <Setup registry={ registry } />;
 	} )
+	.add( 'Set up a new container', ( registry ) => {
```

Comment:
Same as before - let's add a case for the "container already exists" scenario.

Reply:
Added.